app/api/v1/entities/full.rb

```diff
@@ -27,6 +27,20 @@ class IdentityFull < IdentityShallow
             :last_name,
             :ldap_uid
   end
+  
+  class HumanSubjectsInfoFull < HumanSubjectsInfoShallow
+    root 'human_subjects_info', 'human_subjects_info'
+
+    expose  :protocol_id,
+            :nct_number,
+            :hr_number,
+            :pro_number,
+            :irb_of_record,
+            :submission_type,
+            :irb_approval_date,
```

Comment:
Dates should be iso8601 formatted. See https://github.com/sparc-request/sparc-request/blob/go-api/app/api/v1/entities/full.rb#L46